site/content/docs/user/configuration.md

```diff
@@ -281,10 +281,71 @@ nodes:
   image: kindest/node:v1.16.4@sha256:b91a2c2317a000f3a783489dfb755064177dbc3a0b2f4147d50f04825d016f55
 {{< /codeFromInline >}}
 
-[Reference](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster) 
+[Reference](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster)
 
 **Note**: Kubernetes versions are expressed as x.y.z, where x is the major version, y is the minor version, and z is the patch version, following [Semantic Versioning](https://semver.org/) terminology. For more information, see [Kubernetes Release Versioning.](https://github.com/kubernetes/sig-release/blob/master/release-engineering/versioning.md#kubernetes-release-versioning)
 
+### GPU Support
+
+There are two ways to map GPUs in to a KinD cluster. The first is using the `devices` API and the second is using the `extraMounts` API.
+
+#### Using the Devices API
+
+As a pre-requisite you install the [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html) installed on the host.
+
+Using `devices` for GPU support requires Docker v25 or later. See notes on CDI Container Support [here.](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#container-device-interface-cdi-support)
+
+GPU devices can be mapped to Kind node copntainers with the devices API:
+
+All GPUs mapped to a single control-plane:
+
+{{< codeFromInline lang="yaml" >}}
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  devices:
+  - "nvidia.com/gpu=all"
+{{< /codeFromInline >}}
+
+Specific GPUs mapped to specific worker nodes based on index:
+
+{{< codeFromInline lang="yaml" >}}
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+  devices:
+  - "nvidia.com/gpu=0"
+- role: worker
+  devices:
+  - "nvidia.com/gpu=1"
+{{< /codeFromInline >}}
+
+#### Using the Extra Mounts API
+
+GPUs can also be mapped using the `extraMounts` API. This method passes a list of GPUs to inject as volume mounts rather than the environment variable `NVIDIA_VISIBLE_DEVICES`.
+
+Steps to enable this:
+
+1. Add nvidia as your default runtime in /etc/docker/daemon.json
```

Comment:
Assuming that you have a recent version of the NVIDIA Container Toolkit installed, running 
```
sudo nvidia-ctk runtime configure --runtime=docker --set-as-default
```
should set the default runtime.